### ISSUE TITLE
Time and Time Zone options.

### ISSUE THREAD

**Issue by RaveGun** [arthurdent75/SimpleScheduler#15]
Hi,

I just realized that the times that I have setup are off by 2 hours.
It seems, because I have the time zone set in HA to Europe/Berlin, there two hours added to the scheduled times.

In the docker logs there are also the current time + 2h shown.
Like right now:
```172.30.32.2 - - [11/Jun/2020:20:32:37 +0200] "GET / HTTP/1.1" 200 5403 "http://hass.iot:8123/lovelace/default_view" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:78.0) Gecko/20100101 Firefox/78.0"```

The HA's time is fine and is also shown correctly.

Is there a configuration option for the time zone in the addon?

Thank you.

**Comment by arthurdent75** [comment]
No, it takes the timezone from sun.sun 
Take a look there in developement tools.

**Comment by miqmago** [comment]
First of all, thanks @arthurdent75 for this awesome addon! I was up to make my one when I've landed in yours one :)

I've had the same problem, timezone of hassio is setup in `Europe/Brussels` but the addon responds to UTC time. I was really confused until I discovered there was a problem with timezone. I've tried to modify sun entity but haven't found how to change timezone, moreover sun entity shows time in correct timezone (sunset and sunrise are ok).

Seems difficult to understand how to configure timezone, maybe a configuration option in `yaml` could help. Of course still have to dig more on it so I'll find the way, putting on someone else shoes, maybe could help an explanation in readme.

I've also modified [`index.php`](https://github.com/arthurdent75/SimpleScheduler/blob/master/web/index.php#L168) to show the current server time, at least to be sure I make no mistakes until I find how to fix it . Of course it's static but it helps. If you find interesting I can make a pull request:

```php
			<span data-toggle="modal" data-target="#exampleModalLong">Scheduler:</span> <span class="statusbar_span" id="schedulerstatus"><?php echo (is_scheduler_running()) ? "" : "NOT " ?>RUNNING</span>
			<span class="statusbar_span"><?php echo "Internal Time ".date("Y-m-d H:i"); ?></span>
			<span class="statusbar_span"><?php echo "Sunrise ".$sun["sunrise"] ?></span>
			<span class="statusbar_span"><?php echo "Sunset ".$sun["sunset"] ?></span>
```
